File: task-03/hello world/diamond/diamond.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Enter a number: ', (n) => {
  n = parseInt(n);
  for (let i = 0; i < n; i++) {
    console.log(' '.repeat(n - i - 1) + '*'.repeat(2 * i + 1));
  }
  for (let i = n - 2; i >= 0; i--) {
    console.log(' '.repeat(n - i - 1) + '*'.repeat(2 * i + 1));
  }
  rl.close();
});

File: task-03/hello world/diamond/diamond.rs
use std::io;

fn main() {
    let mut input = String::new();
    println!("Enter a number: ");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let n: usize = input.trim().parse().expect("Please enter a number");

    for i in 0..n {
        println!("{}{}", " ".repeat(n - i - 1), "*".repeat(2 * i +
